test/sparse/sparse.jl

```diff
@@ -1642,9 +1642,9 @@ end
 
 # Test temporary fix for issue #16548 in PR #16979. Brittle. Expect to remove with `\` revisions.
 # This is broken by the introduction of RowVector... see brittle comment above.
-#@testset "issue #16548" begin
-#    @test which(\, (SparseMatrixCSC, AbstractVecOrMat)).module == Base.SparseArrays
-#end
+@testset "issue #16548" begin
+    @test_broken which(\, (SparseMatrixCSC, AbstractVecOrMat)).module == Base.SparseArrays
```

Comment:
> a disambiguity method that I added to Sparse

Correct. That there are now two matching methods rather than one is what caused the existing test to fail:
```julia
julia> which(\, (SparseMatrixCSC, AbstractVecOrMat))
ERROR: method match is ambiguous for the specified argument types
Stacktrace:
 [1] which(::Any, ::Any) at ./reflection.jl:715

julia> methods(\, (SparseMatrixCSC, AbstractVecOrMat))
# 2 methods for generic function "\":
\(::SparseMatrixCSC, ::RowVector) in Base.SparseArrays at sparse/linalg.jl:875
\(A::SparseMatrixCSC, B::Union{AbstractArray{T,1},AbstractArray{T,2}} where T) in Base.SparseArrays at sparse/linalg.jl:855
```
Best!